### ISSUE TITLE
Home 2D selection

### ISSUE THREAD

**Issue by XDKRGL** [CybertownRevival/ctr#218]
When moving from say Virtual worlds to 9th Dimension or other colonies that the block_map and 2d images differ we need to force a different settle page where they  must update 2D house icon to that of the specific colony. 

![Screenshot_141](https://github.com/CybertownRevival/ctr/assets/2515831/ed81c6ad-2847-4f0f-ab03-c821c4f4c81b)
Case in point this user moved from VW to 9th Dimension and retained his 2D house icon from VW that does not exist in the 9D assets

his house icon is calling up 
https://www.cybertownrevival.com/assets/img/map_themes/desert/block/Picon2D010.gif
"Picon2D006" is the last choice for desert map

**Comment by smile0711** [comment]
So we need a check for icon, and if not in range, select new icon page.

**Comment by XDKRGL** [comment]
correct, secondary measure allowing admins to update users home in case of this member who has quit otherwise this icon will remain dead on block map.
